pom.xml

```diff
@@ -343,6 +343,17 @@
             <version>2.1</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-artifact</artifactId>
+            <version>3.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
```

Comment:
@gumbelmj The architect fully agrees with the above.

* Testing private methods is entirely unacceptable
 * The need for doing so simply indicates bad design choices 
 * As a consequence in this case but also in general:
   * We should not add any new mock framework to Rultor ever, see http://www.yegor256.com/2014/09/23/built-in-fake-objects.html